my_project/my_app/views.py

```diff
@@ -1,23 +1,92 @@
 """
 Create your views here.
 """
+import json
+from hotels import find_express_deal
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status  # pylint: disable=unused-import
 
 
-class Sum(APIView):
+class BusinessLogic(APIView):
     """
-    Sample view class.
+    This is an example of adding/modifying slots using business logic, using
```

Comment:
I suggest briefly describing the competency design. For example, is it a informational/confirmational, which slots does it have, what are the common conversational scenarios. I think it helps people understand the example better.